java/api/src/main/java/io/ray/api/id/PlacementGroupId.java

```diff
@@ -0,0 +1,48 @@
+package io.ray.api.id;
+
+import java.io.Serializable;
+import java.nio.ByteBuffer;
+import java.util.Arrays;
+import java.util.Random;
+
+public class PlacementGroupId extends BaseId implements Serializable {
```

Comment:
Does `PlacementGroupId` need to be exposed in API?

Reply:
No, I moved it to runtime now.